storm-client/src/jvm/org/apache/storm/daemon/worker/Worker.java

```diff
@@ -246,7 +248,19 @@ public void start() throws Exception {
                             }
                         }
                     });
-              
+
+                workerState.checkForUpdatedBlobsTimer.scheduleRecurring(0,
+                        (Integer) conf.getOrDefault(Config.WORKER_BLOB_UPDATE_POLL_INTERVAL_SECS, 10), new Runnable() {
+                            @Override public void run() {
+                                try {
+                                    LOG.info("Checking if blobs have updated");
+                                    updateBlobUpdates();
+                                } catch (IOException e) {
+                                    e.printStackTrace();
```

Comment:
Can we log this instead of printing it to stderr?  Also it might be nice to have a comment explaining why you are ignoring the exception.